NoteFind/Alerts/ErrorAlerts.m

```diff
@@ -0,0 +1,111 @@
+//
+//  ErrorAlerts.m
+//  NoteFind
+//
+//  Created by Khloe Wright on 7/22/22.
+//
+
+#import "ErrorAlerts.h"
+
+@implementation ErrorAlerts
+
++ (void)loginFailure:(LoginViewController *)loginVC {
+    UIAlertController *alert = [UIAlertController alertControllerWithTitle:@"login failed" message:@"your email or password was entered incorrectly. please try again" preferredStyle:(UIAlertControllerStyleAlert)];
```

Comment:
For any string that will be displayed to the user, wrap it in `NSLocalizedString(...,...)` https://developer.apple.com/documentation/foundation/nslocalizedstring

By default this will just return the string you've specified but its a good practice for when you'd like to localize your app for other languages besides english

Can you update the action titles and buttons throughout this file?